result.go

```diff
@@ -132,7 +132,8 @@ func (r *Result) RootObjectSchemata() []*spec.Schema {
 }
 
 // FieldSchemata returns the schemata which apply to fields in objects.
-// nolint: dupl
+//
+
```

Comment:
Extra empty line?

Reply:
yes we do, normally. Let me remove that extra empty line.